index.html

```diff
@@ -464,6 +467,9 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 be percent-encoded if they are valid token characters except "%",
 and when using percent-encoding, uppercase hex digits MUST be
 used.</p>
+<p class=note>The "hq" ALPN ID is defined for the QUIC protocol in the <a
+   href="https://tools.ietf.org/html/draft-ietf-quic-http-11#section-9.1">HTTP over QUIC Internet Draft</a>.
```

Comment:
We recently changed this:
- https://github.com/w3c/navigation-timing/issues/71
- background: https://lists.w3.org/Archives/Public/ietf-http-wg/2018JanMar/0204.html 

We should probably capture the resolution from that discussion in this note? 

Otherwise, LGTM!

Reply:
Theoretically, that has not changed, it's just that Chrome is using a different string for G-QUIC, instead of publishing it as the IETF variant, which "owns" the ALPN ID.

I wonder if we need to also add some liberty on that front for experimental protocols. Maybe something like: "In case the network protocol used is an experimental one and not defined in the above registry, user agents MAY also use other descriptive strings."

Reply:
Yeah, that makes sense. In fact, perhaps invert it and lead with note about experimental protocols, citing QUIC as one example.

Reply:
I think the bit about experimental protocols needs to be normative. Maybe I can add it around the HTTP/0.9 and 1.0 bits. (I'm also thinking it'd make for a long paragraph, so maybe I should split it up into a list somehow)

Reply:
(belatedly) added the above. PTAL